README.md

```diff
@@ -83,3 +84,13 @@ class Command(BaseCommand):
                         old_cache.delete(actual_key)
 
 ```
+
+# Scheduler related usage
+You can use some extra functionality provided by this plugin, if secure cache settings are not added in the `default` settings, please make sure that you specify your secure cache settings name by setting `DJANGO_REDIS_SECURE_CACHE_NAME` settings. To enable these feature, this simply by adding `@secure_redis.rq_job` decorator on top of your task. By doing this you can get use of:
```

Comment:
This library also provides additional functionality for that encrypts the payload of RQ jobs. To make use of this functionality, first ensure that `DJANGO_REDIS_SECURE_CACHE_NAME` is defined in your settings (if not set, this setting will default to using the `default` cache). Once configured, replace all instances of the `django_rq.job` decorator with `@secure_redis.rq_job`. The `@secure_redis.rq_job` provides the following functionality: